imap_data_access/cli.py

```diff
@@ -265,13 +282,18 @@ def main():
 
     # Upload command
     parser_upload = subparsers.add_parser(
-        "upload", help=upload_help, description=upload_help
+        "upload", help=upload_help, description=help_menu_for_upload
     )
     parser_upload.add_argument("file_path", type=Path, help=file_path_help)
     parser_upload.set_defaults(func=_upload_parser)
 
     # Parse the arguments and set the values
-    args = parser.parse_args()
+    try:
+        args = parser.parse_args()
+    except Exception:
+        print("Please provide input parameters.")
+        sys.exit(0)
```

Comment:
Thanks for adding this! Two nitpicks - you should catch very specific exceptions (in this case, a typeerror is the one we're concerned about) and rather than a print statement and then an exit, you can just raise a custom exception. (you can see a list of exception types [here](https://docs.python.org/3/library/exceptions.html), a "valueerror" is a nice generic one for custom exceptions in many cases. It's best to pick an exception that makes sense if possible, or even write a custom exception.)

```suggestion
    except TypeError:
        raise ValueError("Please provide input parameters, or use `-h` for more information.")
```

So, this will still raise an error, which we want because the code is in a bad state and can't continue. But, it will be a super clear and simple error, with your custom text.  

More info on exceptions: https://realpython.com/python-raise-exception/

Reply:
For clis it is nicer to do parser.exit() so you don't get the full traceback and get a failing error code with the cli name etc.

See below for an example of that.

Reply:
I see, I see. Let me know if what I changed it to is what is wanted and if it adequately conveys the problem raised.